airflow/api_fastapi/execution_api/datamodels/xcom.py

```diff
@@ -17,10 +17,21 @@
 
 from __future__ import annotations
 
+import sys
+from typing import Any
+
 from pydantic import JsonValue
 
 from airflow.api_fastapi.core_api.base import BaseModel
 
+if sys.version_info < (3, 12):
+    # Since 3.9 doesn't have a proper TypeAlias, this causes a hiccough:
```

Comment:
If we submit a bugreport to Cadwyn, I can try fixing that within there.

Reply:
Reported https://github.com/zmievsa/cadwyn/issues/262